ui/index.d.ts

```diff
@@ -140,20 +140,34 @@ export abstract class SubscriptionTreeItemBase extends AzExtParentTreeItem {
     constructor(parent: AzExtParentTreeItem, subscription: ISubscriptionContext);
 }
 
+/**
+ * Loose type to use for T1 and T2 versions of "@azure/ms-rest-js".  The Azure Account extension returns
+ * credentials that will satisfy both T1 and T2 requirements
+ */
+export type AzExtServiceClientCredentials = AzExtServiceClientCredentialsT1 | AzExtServiceClientCredentialsT2;
```

Comment:
If it satisfies both should it be this?
```suggestion
export type AzExtServiceClientCredentials = AzExtServiceClientCredentialsT1 & AzExtServiceClientCredentialsT2;
```

(Or maybe it's `AzExtGenericCredentials` that should be the union)

Reply:
Absolutely right.  Thanks, this is why you don't code during stand up lmao